nav2_costmap_2d/src/layered_costmap.cpp

```diff
@@ -99,6 +99,12 @@ void LayeredCostmap::resizeMap(
   }
 }
 
+bool LayeredCostmap::isOutofBounds(double robot_x, double robot_y)
+{
+  unsigned int mx, my;
+  return costmap_.worldToMap(robot_x, robot_y, mx, my);
```

Comment:
Shouldn't this be `return !costmap_.worldToMap(robot_x, robot_y, mx, my);` ?

because:
```

   * @return True if the conversion was successful (legal bounds) false otherwise
   */
  bool worldToMap(double wx, double wy, unsigned int & mx, unsigned int & my) const;
```

Reply:
Agreed. I’m not sure how that didnt appear in my testing logs. Let me check my local branch and make sure I didnt miss a commit. 

From a quick look I agree, let me make sure next week that that’s indeed the case.

Reply:
Fixed. @mkhansen-intel tried to create new branch on phone and pr that but somehow committed directly to master (stuck on train for 2 hours waiting... figured I’d get a start on the day... wtf github). I can revert it and PR if you like.